truss/tests/test_model_inference.py

```diff
@@ -193,6 +193,29 @@ def test_requirements_file_truss():
         assert response.json() is False
 
 
+@pytest.mark.integration
```

Comment:
Since this is a totally new use case and structure, I'd suggest putting this in a "new section" at the end of the test file.

Reply:
will move it out as a new test file